src/dsp_tools/commands/xmlupload/models/deserialise/deserialise_value.py

```diff
@@ -202,7 +204,7 @@ class XMLBitstream:
     """
 
     value: str
-    permissions: Optional[str]
+    permissions: Optional[str] = None
```

Comment:
Since you are already changing code you could also change the typing here into `str | None`.